pkg/controller/core/workload_controller.go

```diff
@@ -216,8 +219,11 @@ func (r *WorkloadReconciler) reconcileSyncAdmissionChecks(ctx context.Context, w
 		log := ctrl.LoggerFrom(ctx)
 		log.V(3).Info("The workload needs admission checks updates", "clusterQueue", klog.KRef("", cqName), "admissionChecks", queueAdmissionChecks)
 		wl.Status.AdmissionChecks = newChecks
-		err := r.client.Status().Update(ctx, wl)
-		return true, client.IgnoreNotFound(err)
+		if err := r.client.Status().Update(ctx, wl); err != nil {
```

Comment:
The more I look at this, the less value I see.

Users shouldn't be interested about this either.
They just want to know when the admission checks are satisfied. Let's remove this event too.

Reply:
removed https://github.com/kubernetes-sigs/kueue/pull/1271/commits/e27ddfae37502bac202e7688a52f8624a71dfd72